src/modbus-rtu.c

```diff
@@ -1112,6 +1153,40 @@ int modbus_rtu_set_rts_delay(modbus_t *ctx, int us)
     }
 }
 
+int modbus_rtu_set_suppress_echo(modbus_t *ctx, int on)
+{
+    if (ctx == NULL) {
+        errno = EINVAL;
+        return -1;
+    }
+
+    if (ctx->backend->backend_type == _MODBUS_BACKEND_TYPE_RTU) {
+        modbus_rtu_t *ctx_rtu = ctx->backend_data;
+        ctx_rtu->suppress_echo = !!on;
```

Comment:
can't it just take a bool instead of this sort of thing?

Reply:
I took confirmation_to_ignore field as example - which is int but could be a bool, too. I've no strong preference here, let's wait for other comments...

Reply:
I think I used 'int' for confirmation_to_ignore for speed of access vs full size of the structure (memory alignment and padding).